### PR TITLE
Urological supplies order

### PR DESCRIPTION
**Pull request by pmadria**
This is the pull request for changing the service request to device request for urological supplies.

### CODE REVIEW

File: src/main/resources/fhirServer.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 client_id = app-token
-client_secret= #replaceMe#
+client_secret= ddc9eec0-8b37-48b6-84f5-9b7ce3f5c881
```
Comment (yuliumitre) [inline src/main/resources/fhirServer.properties:2]: Is this supposed to be checked in?

File: fhirResourcesToLoad/o2jane_04__device-request-a4217.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    "subject": {
+        "reference": "Patient/pat016"
+    },
+    "occurrenceDateTime": "2016-09-27",
+    "authoredOn": "2016-09-20",
+    "requester": {
+        "display": "Smythe Juliette, MD"
+    },
+    "performer": [
```
Comment (yunwwang) [inline fhirResourcesToLoad/o2jane_04__device-request-a4217.json:23]: Performer is not an array

File: fhirResourcesToLoad/o2jane_04__device-request-a4217.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+    "resourceType": "DeviceRequest",
+    "id": "devreq16-a4217",
```
Comment (yunwwang) [inline fhirResourcesToLoad/o2jane_04__device-request-a4217.json:3]: These elements are required: DeviceRequest.basedOn, DeviceRequest.instantiatesCanonical

File: fhirResourcesToLoad/o2jane_04__device-request-a4217.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        "display": "Smythe Juliette, MD"
+    },
+    "performer": [
+        {
+            "reference": "Practitioner/pra1255"
+        }
+    ],  
+    "extension": [
+        {
+          "url": "http://build.fhir.org/ig/HL7/davinci-crd/R4/ext-insurance.html",
```
Comment (yunwwang) [inline fhirResourcesToLoad/o2jane_04__device-request-a4217.json:30]: Cannot use this extension here

File: src/main/resources/fhirServer.properties (hunk 1 above)
Comment (pmadria) [reply src/main/resources/fhirServer.properties:2]: should I go ahead and remove this?

Comment (yuliumitre) [reply src/main/resources/fhirServer.properties:2]: Yeah, please roll back the change.

Comment (pmadria) [reply src/main/resources/fhirServer.properties]: reverted to normal

File: fhirResourcesToLoad/o2jane_04__device-request-a4217.json (hunk 3 above)
Comment (pmadria) [reply fhirResourcesToLoad/o2jane_04__device-request-a4217.json:3]: not sure if I should add these and if so where?

Comment (yunwwang) [reply fhirResourcesToLoad/o2jane_04__device-request-a4217.json:3]: Please check other DeviceRequest resources

**Pull request merged**